followers.go

```diff
@@ -0,0 +1,120 @@
+package twitter
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+	"strconv"
+)
+
+// FollowerListParams represents the query parameters for a /search/tweets.json
+// request.
+type FollowerListParams struct {
+	UserID              string
+	ScreenName          string
+	Cursor              string
+	Count               int
+	SkipStatus          bool
+	IncludeUserEntities bool
```

Comment:
Should be `ExcludeUserEntities`, so that we will only use this field if set to `true`.